### PR TITLE
Foundation Setup For Manutal Test Scripting

### PR DESCRIPTION
**Pull request by KtorZ**
<p align="right">#258</p>

# Overview

- [ ] I have added `docopt` as a vendor dependency in the `test/manual/scripts`
- [ ] I have added two examples for CLI bash scripts that can be written using `docopt`, making shell script a bit less of a mess.

# Comments

### CODE REVIEW

File: test/manual/scripts/docopts.sh [hunk 1]
```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env bash
```
Comment (KtorZ) [inline test/manual/scripts/docopts.sh:1]: This is imported from the docopt repository, I couldn't resort to have a submodule here, so I just copy/pasted the script and added a LICENSE header.

File: test/manual/scripts/generate-addresses.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+do
+  >&2 echo "Address no: $i"
+	ADDRESS=$(curl -kX POST https://localhost:${ARGS[--port]}/api/v1/addresses \
+  		-H "Accept: application/json; charset=utf-8" \
+  		-H "Content-Type: application/json; charset=utf-8" \
+  		--cert ${ARGS[--client]} \
+  		-d "{
+		  \"accountIndex\": ${ARGS[--account]},
+		  \"walletId\": \"${ARGS[--wallet]}\"
+      }" | jq .data.id)
```
Comment (piotr-iohk) [inline test/manual/scripts/generate-addresses.sh:28]: `jq .data.id` - that's cool! :+1:

**Review by piotr-iohk — APPROVED** [review]
:+1: :+1: :+1:

File: test/manual/scripts/generate-addresses.sh (hunk 2 above)
Comment (KtorZ) [reply test/manual/scripts/generate-addresses.sh:28]: Not sure if `jq` is installed by default on most Mac OS and Linux distributions but that's definitely a must-have when working with JSON :sweat_smile:

**Pull request merged**